scripts/common.sh

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env bash
 
-# shellcheck disable=SC2154
+# shellcheck disable=SC2154,SC2155
```

Comment:
I would not do this. I fear we will get bad and misleading error-messages: <https://www.shellcheck.net/wiki/SC2155>